app/assets/javascripts/application.js

```diff
@@ -57,7 +42,7 @@ $(document).ready(function(){
 
     $.ajax({
       method: "POST",
-      url: baseUrl + "/organizations/filter",
+      url: BASEURL + "/organizations/filter",
```

Comment:
You shouldn't need `BASEURL` because you're making an AJAX request from within your organization